### PR TITLE
completes xrpl keyword mapping recs for xrpl.org

### PR DESCRIPTION
**Pull request by AKCodez**
Adds changes requested from [spreadsheet](https://docs.google.com/spreadsheets/d/1dsbZ1hyKVSQpEZAmBnEXi31rUoLmyH_smmwgLRdA4R8/edit#gid=447031354)

### CODE REVIEW

File: docs/concepts/payment-types/direct-xrp-payments.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 html: direct-xrp-payments.html
 parent: payment-types.html
 seo:
-    description: Direct XRP payments are the simplest way to send value in the XRP Ledger.
+    title: Direct XRP Payment Fundamentals
```
Comment (amarantha-k) [inline docs/concepts/payment-types/direct-xrp-payments.md]:
```suggestion
    title: Direct XRP Payments
```
This topic is already under concepts and Payment types, so "Fundamentals" in the title seems redundant.

File: docs/tutorials/index.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 ---
 seo:
-    description: The XRP Ledger tutorials walk you through the steps to learn and get started with the XRP Ledger and to use the ledger for advanced use cases.
+    title: Crypto Wallet and Blockchain Development Tutorials
+    description: Learn the basics on how to get started building on the XRP Ledger with these helpful crypto wallet and blockchain tutorials for developers.
 ---
-# Tutorials
+# Crypto Wallet and Blockchain Developer Tutorials
```
Comment (amarantha-k) [inline docs/tutorials/index.md]: Can we keep the page title as-is?

File: docs/tutorials/index.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
 ---
 seo:
-    description: The XRP Ledger tutorials walk you through the steps to learn and get started with the XRP Ledger and to use the ledger for advanced use cases.
+    title: Crypto Wallet and Blockchain Development Tutorials
+    description: Learn the basics on how to get started building on the XRP Ledger with these helpful crypto wallet and blockchain tutorials for developers.
```
Comment (amarantha-k) [inline docs/tutorials/index.md]: The tutorials include both beginner and advanced topics, so recommend updating the description to reflect that.

**Review by amarantha-k — CHANGES_REQUESTED** [review]

File: docs/tutorials/index.md (hunk 3 above)
Comment (AKCodez) [reply docs/tutorials/index.md]: i removed "the basics on"

**Review by amarantha-k — APPROVED** [review]

**Pull request merged**